build.gradle

```diff
@@ -15,7 +15,7 @@ plugins {
 }
 
 group = 'uk.gov.hmcts.et'
-version = '0.0.1'
+version = '2.2.0'
```

Comment:
Is this version correct? it might be better to just remove this change if we're not clear on numbering

Reply:
Good point. removed